### PR TITLE
text aligned to center

### LINKED ISSUE #3898: align text to center
### Current Behavior

currently the text is aligned to right

### Desired Behavior

align text to the center

### Screenshots / Mockups

Screenshot 2023-03-06 at 12 03 21 AM

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by ShravaniAK**
**Description**

This PR fixes #3898 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by sudhanshutech — APPROVED** [review]
good work

**Review by UtkarshMishra12 — APPROVED** [review]
LGTM Thanks @ShravaniAK 🥇🚀

File: src/sections/Meshery/meshery.style.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const MesheryWrapper = styled.section`
     .callout{
         margin: auto;
         margin-top: 5rem;
-        text-align: right;
+        text-align: center;
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/meshery.style.js:69]: This change should happen only when the resolution is less than `768px`. Please write a media query to fix this issue.

**Comment by l5io** [comment]
🚀 Preview for commit 92c0e4dcd0cc4aefb293766399bbaed8ebf8bd01 at: https://640d50f3dcf7467faa4b2e2f--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
One last thing, update the `margin-top` value to be `4rem` for media query `max-width: 32rem`, currently it creates a lot of white space.
Screenshot 2023-03-12 at 11 38 38 AM

**Comment by l5io** [comment]
🚀 Preview for commit 095bc3175cdfeecbd630d110e43afa061b1f3761 at: https://640d72d57eccd43bd10f508c--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Nicely done, @ShravaniAK 👌🏼

**Pull request merged**